gpkit/geometric_program.py

```diff
@@ -415,9 +415,10 @@ def _solve_sweep(self, printing, skipfailures, allownonoptimal):
         self.presweep = self.last
 
         if len(self.sweep) == 1:
-            sweep_grids = np.array(self.sweep.values())
+            sweep_grids = np.array(list(self.sweep.values()))
```

Comment:
Profiled, this takes ~0% of the solve time, so I'm tempted to leave it be for readability.